odoo/tests/common.py

```diff
@@ -537,8 +537,23 @@ def executable(self):
 
         raise unittest.SkipTest("Chrome executable not found")
 
+    def _spawn_chrome(self, cmd):
+        pid = os.fork()
+        if pid != 0:
+            return pid
+        else:
+            if os.name == 'posix' and platform.system() != 'Darwin':
+                # since the introduction of pointer compression in Chrome 80 (v8 v8.0),
+                # the memory reservation algorithm requires more than 8GiB of virtual mem for alignment
+                # this exceeds our default memory limits.
+                # OSX already reserve huge memory for processes
+                resource.setrlimit(resource.RLIMIT_AS, (resource.RLIM_INFINITY, resource.RLIM_INFINITY))
+                with open(os.devnull, 'wb', 0) as stdout_replacement:
+                    os.dup2(stdout_replacement.fileno(), sys.stderr.fileno())
```

Comment:
- The dup2 is also needed on Darwin, I guess
- Any reason to call it `stdout_replacement` and then use it for stderr only? ;-)

Reply:
good catch, I tested with stdout :smiley: 
And yes I have to separate the Darwin stuff

Reply:
Yes, dup2 should be called on all platforms.